.github/workflows/validate-licenses.yaml

```diff
@@ -0,0 +1,36 @@
+name: Check licenses.d2iq.yaml
+on:
+  pull_request: {}
+  workflow_dispatch: {}
+  push:
+    tags:
+      - v*
+    branches:
+      - main
+jobs:
+  check-license-yaml:
+    runs-on: ubuntu-latest
+    name: Check licenses.d2iq.yaml
+    env:
+      BLOODHOUND_VERSION: v0.8.0
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Checkout dkp-infra-tools
+        uses: actions/checkout@v3
+        with:
+          repository: mesosphere/dkp-infra-tools
+          ref: licenses/v0.0.5
```

Comment:
nit. maybe parametrize also this version as Bloodhound?

Reply:
This doesn't seems to be possible - https://github.com/orgs/community/discussions/26529